x/bounty/client/cli/flags.go

```diff
@@ -7,5 +7,11 @@ const (
 	FlagDeposit           = "deposit"
 	FlagSubmissionEndTime = "submit-end"
 
+	FlagFindingTitle         = "title"
+	FlagFindingDesc          = "desc"
+	FlagProgramID            = "program_id"
```

Comment:
Make the flags consistent using "_" or "-" ?

Reply:
sure, fixed